python/ray/parameter.py

```diff
@@ -208,10 +202,3 @@ def _check_usage(self):
         if self.redirect_output is not None:
             raise DeprecationWarning(
                 "The redirect_output argument is deprecated.")
-
-        if self.use_pickle:
-            assert (version.parse(
-                np.__version__) >= version.parse("1.16.0")), (
```

Comment:
This needs to be left in. We should print a warning that on numpy < 1.16, serialization of arrays will be significantly slower.

Reply:
We already require >= 1.16.0 in setup.py so I'll change this to a warning instead of an assertion.